### PR TITLE
Loaded Command Preview Pull Request

### PR DESCRIPTION
**Pull request by DanielWap**
Please review our Pull Request.
We added functionality to Preview Command in Command Manager.

### CODE REVIEW

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java [hunk 1]
```diff
@@ -71,17 +82,34 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
...
 
+	public void updateCurrentCommandPreview() {
+		clearCurrentCommandPreview();
+		DriverCommand command = commandManager.getCurrentCommand();
+		if (command != null) {
+			command.execute(previewAdapter);
+		}
+	}
+
+	private void clearCurrentCommandPreview() {//Function for clearing screen
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java]: unnecessary comment

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindowCommandChangeObserver.java [hunk 2]
```diff
@@ -4,20 +4,21 @@
...
+        super();
+        this.commandManagerWindow = commandManagerWindow;
+    }
 
-	public String toString() {
-		return "Current command change observer for command manager window";
-	}
+    public String toString() {
+        return "Current command change observer for command manager window";
+    }
```
Comment (kowallus) [inline src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindowCommandChangeObserver.java]: spaces instead of tabs

**Review by kowallus — CHANGES_REQUESTED** [review]
some minor fixes.

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java (hunk 1 above)
Comment (DanielWap) [reply src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java]: Fixed in latest commit

File: src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindowCommandChangeObserver.java (hunk 2 above)
Comment (DanielWap) [reply src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindowCommandChangeObserver.java]: Fixed in latest commit

Comment (kowallus) [reply src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindowCommandChangeObserver.java]: not really. Maybe your IDE makes spaces when saving file

**Review by kowallus — CHANGES_REQUESTED** [review]
spaces instead of tabs!

**Pull request merged**